projects/js-packages/publicize-components/src/components/add-connection-modal/connect-page/style.module.scss

```
@@ -0,0 +1,53 @@
+.example-wrapper {
+	display: grid;
+	grid-auto-columns: minmax(0, 1fr);
+	grid-auto-flow: column;
+	font-size: 15px;
+	margin-top: 0.4rem;
+	min-height: 300px;
+
+	&.small {
+		grid-auto-flow: row;
+		grid-gap: 2rem;
+		height: auto;
+		min-height: unset;
+	}
+
+	img {
+		max-height: 520px;
+		max-width: 100%;
+	}
+
+	> div {
+		padding: 0.4rem;
+		
+		&:has(img) {
+			margin-left: auto;
+			margin-right: auto;
```

Let us make it RTL aware.

```suggestion
			margin-inline-start: auto;
			margin-inline-end: auto;
```